### ISSUE TITLE
Make Progress bar in Home Screen match the what's in Tools

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#3296]
The progress bar in the tool card works properly - starts black on white, then centers on the progress bar and becomes white on dark blue. Make this same functionality happen on the tool card of the home screen.

**Comment by richmahn** [comment]
- [ ] [2] Make progress bar on home screen match tools
- [ ] [2] Test progress bar on home screen behaves like tools
- [ ] [1] PR Review for #3296

**Comment by cckozie** [comment]
After clicking on the Home icon while in an alignment project, the percent complete displayed on the tool card on the home page reflects the % complete when the project was opened, not when it was closed.

**Comment by richmahn** [comment]
@cckozie @benjore This is a known problem which already was occurring before I re-aligned the progress bar to be like the tool bar. As discussed in the PR by us developers, we felt this should be a separate issue. If you feel it should be fixed in this issue, let me know.

**Comment by cckozie** [comment]
@richmahn The title of this issue is "Make Progress bar in Home Screen match the what's in Tools". Since the % displayed is not the same, in my mind this is not a match.
@benjore Do you see it differently?

**Comment by richmahn** [comment]
@cckozie I'm the one that even made Ben write this up just to get the colors we decided in the bar match, where < 20% the nume is black and in the middle of the area, otherwise floats in the filled part of the bar. Yet as I said, even when developing we noticed that this home screen bar never updates unless you visit the Tool card page, then for some reason it updates. Will try to see if we can get that behavior to happen on the home page as well.
